### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by monika1281** [recodehive/Opensource-practice#39] [General]
Being very new to open source i dont know how to choose an organization because i feel i have very few skills which aren't worth contributing i guess?

**Comment by Ali-Hassan2** [comment]
@monika1281 I think the best way to choose an organization is to first look at your own interests and the technologies you enjoy working with. Once you’re clear on that, you can explore organizations in that domain, check out their repositories, go through open issues, and see if there’s something you can realistically contribute to. Even starting with small issues or documentation helps you understand the workflow, and over time you’ll get more comfortable picking bigger challenges.

**Comment by georgiggg** [comment]
@monika1281 Honestly, that’s a really normal feeling when you’re just starting open source, and you definitely don’t need to be super skilled to contribute. I’d choose an organization based on something you actually find interesting and that has beginner-friendly issues, since even small things like fixing docs, typos, or simple bugs still matter. Everyone starts somewhere, and a lot of people build their skills by making those smaller contributions first.
